dex/testing/dcr/harness.sh

```diff
@@ -28,20 +28,37 @@ TRADING_WALLET2_PORT="19582"
 WAIT="; tmux wait-for -S donedcr"
 
 NODES_ROOT=~/dextest/dcr
+export NODES_ROOT
+
 if [ -d "${NODES_ROOT}" ]; then
   rm -R "${NODES_ROOT}"
 fi
 mkdir -p "${NODES_ROOT}/alpha"
 mkdir -p "${NODES_ROOT}/beta"
 mkdir -p "${NODES_ROOT}/harness-ctl"
 
+MINE=1
+# Bump sleep up to 3 if we have to mine a lot of blocks, because dcrwallet
+# doesn't always keep up.
+MINE_SLEEP=3
```

Comment:
6x slower seems like an extreme change.  Was this empirically determined?  I rarely had an issue with 0.5 sec, but I ack that my machine might have been fast enough for wallet to keep up.

Reply:
I know I checked at 1 second and consistently ran into the issue. Regardless, this will almost never be used since we'll be loading from file now.

Reply:
Oh but it keeps getting used in `mine-alpha` even after the chain is set up, which is sort of a pain, and probably makes it so trade_simnet_test.go will always need to be started with the archive backed harness or the tests will be even more irritating to run.

Reply:
I consider the `MINE=1` mode to be used almost exclusively for archive generation now.